test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -118,11 +118,6 @@ def statvfs_called():
 
         wait_until(10, 1, statvfs_called)
 
-        # these can sometimes happen during startup before any tenants have been
-        # loaded, so nothing can be evicted, we just wait for next iteration which
-        # is able to evict.
-        self.neon_env.pageserver.allowed_errors.append(".*WARN.* disk usage still high.*")
-
```

Comment:
This can no longer happen because of #4399 or earlier #4372 which stalls the background jobs until loading is done.